### PR TITLE
Shiva/eng 7662 charge gas fee on elrond instead of 0

### PR DESCRIPTION
**Pull request by shivamangina**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
     "bignumber.js": "9.0.1",
     "chai": "4.2.0",
     "dotenv": "8.2.0",
-    "joi": "17.6.0"
+    "joi": "17.6.0",
+    "lodash": "^4.17.21"
```
Comment (thakursachin467) [inline package.json:23]: @shivamangina What's the purpose of adding lodash lib here? I don't see it's usage

File: index.js [hunk 2]
```diff
@@ -232,11 +232,36 @@ async function sendTransaction({to, amount, network, keyStore, password, tokenId
...
+    if (network === "main") networkDetails = config.networks.main;
+    const provider = new ProxyNetworkProvider(networkDetails.provider, {
+      timeout,
+    });
+    const networkConfig = await provider.getNetworkConfig();
+  
+    return {
+      gasLimit,
+      gasCostInCrypto: Number(
+        _toDecimal(gasLimit * networkConfig.MinGasPrice, decimals)
```
Comment (thakursachin467) [inline index.js:253]: @shivamangina From where are we getting this `gasLimit` here?

File: package.json (hunk 1 above)
Comment (shivamangina) [reply package.json:23]: The lodash is being used in the code already. but is missing in the package.json so, I added.
```
async function sendTransaction({to, amount, network, keyStore, password, tokenIdentifier, decimals}) {
    const {error} = sendTransactionParams.validate({to, amount, network, keyStore, password, options: {tokenIdentifier, decimals}});
    if (error) {
        throw new Error(`${error.message}`)
    }
    try {
        //Set network
        let networkDetails = config.networks.testnet;
        if (network === 'main') networkDetails = config.networks.main;
        if(!_.isNumber(decimals)) decimals=18;

 ```

File: index.js (hunk 2 above)
Comment (shivamangina) [reply index.js:253]: We are getting from this function 

```
 { payload, payment, gasLimit } = _getPayload({
      identifier: tokenIdentifier,
      amount,
      decimals,
    });
 ```
    
   this is existing function which is being used for setting gasLimit in the sendTransaction.

**Review by thakursachin467 — APPROVED** [review]

**Pull request merged**